### DISCUSSION TITLE
Content width / 内容宽度

### DISCUSSION THREAD

**Discussion by Mister-Hope** [vuepress-theme-hope/vuepress-theme-hope#3742] [Show and tell]
The theme has a reasonable content width.

For UI design, websites will avoid to render content too width.

When the content is too wide, the following disadvantages appears:

- reading difficulty to find the next line (You likely to forget the first word of current line when reading to line end)
- heads moving is required besides sights. (like you take some drugs)
- ...

See some common website as example, they all stop expanding when content is around 1000px:

- https://github.com/
- https://www.bing.com/search?q=test
- https://lessig.medium.com/chatgpt-or-how-i-learned-to-stop-worrying-and-love-ai-242f181723af

---

主题具有合理的内容宽度。

对于 UI 设计，网站会避免将内容渲染得太宽。

当内容太宽时，会出现以下缺点：

- 阅读时很难找到下一行 （读到行尾时你很有可能已经忘记当前行的第一个字词）
- 只动眼阅读会很困难，必须伴随摇头。 (就像磕了药一样)
- ...

以国内一些常见网站为例，它们都在 1000 像素左右就停止扩展宽度了：

- https://weibo.com/
- https://www.zhihu.com/
- https://www.baidu.com/s?wd=%E6%B5%8B%E8%AF%95
- https://mp.weixin.qq.com/s?__biz=MzIyNjU0OTcwNA==&mid=2247536107&idx=1&sn=63b02ad3490de7c53030c46dd16a21a8&chksm=e86cc74cdf1b4e5af9ea239c10e5d2f231b062a5e1c0d6acdb59a6ddd66310d02dc71349e751&mpshare=1&scene=23&srcid=0811UVJq6SzOm3wTmKQGHa0d&sharer_shareinfo=f72a616ca2cd4223948198e3cdacdb67&sharer_shareinfo_first=c7055e92d1ca3c22c68abde2fc41eb18#rd

---

If you think that placing sidebar and toc at the screen edge is better, let's straight to the point: you know poor things about UI design, or there are some issues with your aesthetic.

如果你认为把侧边栏和目录放在屏幕边缘更好，那我们就开门见山吧：你对UI设计的了解很差，或者你的审美有一些问题。

![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/33315834/0b255558-45ba-4eae-b2e3-cee74d62ca5c)

However, if you insist, you may try adding the following style:

不过，如果你坚持的话，你可以尝试添加以下样式：

```scss
// .vuepress/styles/index.scss
@use "@sass-palette/hope-config";

@media (min-width: hope-config.$pc) {
  html {
    --content-width: calc(100vw - 40rem)!important;
  }
}
```

**Comment by bihell** [comment]
我喜欢下面这个宽的
